modules/lang-painless/src/main/java/org/elasticsearch/painless/PainlessPlugin.java

```diff
@@ -140,6 +143,24 @@
         filter.add(filterWhitelist);
         map.put(FilterScript.CONTEXT, filter);
 
+        List<Whitelist> aggregation = new ArrayList<>();
```

Comment:
At this point we should have something like `getRuntimeFieldWhitelist` for this API since a lot of this config code is the same.

Reply:
Agreed, I would prefer to do this as a follow up PR. I even wonder if this should maybe be moved elsewhere. The entirety of including these contexts as part of the module is a bit of an afterthought, though I understand doing them through SPI doesn't make sense here given the dependency hierarchy.

Reply:
Well we should cleanup the whitelisting code. Next PR is fine, but if we wanna do a more fundamental change then I'd prefer the wl code cleanup happen first.